src/forms/TemplateForm.css

```diff
@@ -0,0 +1,8 @@
+.template {
```

Comment:
General Question: Why did this PR touch template form?

Reply:
removed and added to gitignore